R/format_ay.R

```diff
@@ -17,3 +17,24 @@ format_ay <- function(year) {
   }
   sub("(.{4})(.*)", "\\1/\\2", year)
 }
+
+#' Undo academic year formatting
+#'
+#' This function converts academic year variables back into 201617 format.
+#'
+#' It accepts character arguments.
+#'
+#' @param year Academic year
+#' @return Unformatted year
+#' @export
+#' @examples
+#' format_ay_reverse("2016/17")
+
+format_ay_reverse <- function(year) {
+  if (!grepl("^\\d{4}/\\d{2}.*", year)) {
+    stop("year parameter must be a seven digit string e.g. '2016/17'")
```

Comment:
Nitpicky: Do we need to be any more specific in the error about the seven digit string? 

Maybe "Year input must be formatted as 'YYYY/YY' "? 

I know we have the example but the message itself of 'seven digit string' isn't that specific if that makes sense?